### PR TITLE
Use CSS to style UI

### PR DESCRIPTION
**Pull request by yuya-oc**
Before submitting, please confirm you've
 - [x] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
 - [x] completed [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/)
 - [x] executed `npm run lint:js` for proper code formatting

Please provide the following information:

**Summary**
Use CSS to style UI.

A part of prior work for #457 to simplify layout.

Styles were divided to React's inline `style` and CSS. It's confusing, so I want to migrate to CSS via `style-loader` and `css-loader`. (Setting page is not migrated yet because it's complex)

**Issue link**
N/A

**Test Cases**
Functionality tests are automated.

Need to check:
- Tab bar
- Dialogs
- Layout and color

**Additional Notes**
https://circleci.com/gh/yuya-oc/desktop/260#artifacts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
     "babel-preset-react": "^6.24.1",
     "chai": "^3.5.0",
     "cross-env": "^4.0.0",
+    "css-loader": "^0.28.1",
```
Comment (jasonblais) [inline package.json]: can we add NOTICE.txt entry for this?

File: package.json [hunk 2]
```diff
@@ -56,6 +57,7 @@
     "mocha-circleci-reporter": "0.0.2",
     "npm-run-all": "^4.0.2",
     "spectron": "~3.6.2",
+    "style-loader": "^0.17.0",
```
Comment (jasonblais) [inline package.json]: can we add NOTICE.txt entry for this?

**Comment by jasonblais** [comment]
Tested on Windows 10 Home
 - Tab bar
 - Dialogs
 - Layout and color

All looks good to me. Tested on various screen resolutions.

@yuya-oc Can you help check the tab bar looks good on Mac, particularly the "+" sign? We've had it vertically misaligned a few times before.

Also, are these the two libraries we're adding in this PR? If so, I'd be happy to add the NOTICE.txt entries
 - https://github.com/webpack-contrib/style-loader
 - https://github.com/webpack-contrib/css-loader

**Comment by jasonblais** [comment]
Thanks!

**Comment by yuya-oc** [comment]
I tested on macOS. Unfortunately, it's misaligned in v3.7.0 as well... I will prepare another PR.

**Pull request merged**